Gems/Atom/RHI/Code/Source/RHI/PipelineStateCache.cpp

```diff
@@ -386,15 +402,17 @@ namespace AZ::RHI
         switch (descriptor.GetType())
```

Comment:
This switch is duplicated in `MultiDevicePipelineState::Init()`. Just calling `pipelineState->Init()` here should work as well.